app/components/file-header/template.emblem

```diff
@@ -120,22 +120,12 @@
             a click="displayOWASPDetails" class="uppercase-text"
               | Owasp Details
     .column.content-right
-      .card-button
-        .columns
-          .column.card-button
-            if organization.selected.isTrial
-              button.is-primary.margin-top20 click={(perform openRequestAccessModal)}
-                = fa-icon "download"
-                | &nbsp; #{ t "pdfReport"}
-            else
-              button.mp-pdf-report click="getPDFReportLink" disabled=isDownloadingReport
-                if isDownloadingReport
-                  .fa-font-size
-                    i.fa class="fa-spinner fa-spin"
-                    | &nbsp;
-                else
-                  = fa-icon "download"
-                | &nbsp; #{ t "pdfReport"}
+      if organization.selected.isTrial
+        button.is-primary.margin-top-5 click={(perform openRequestAccessModal)}
+          = fa-icon "download"
+          | &nbsp; #{ t "pdfReport"}
+      else
+        = file/report-btn file=file showCopyPasswordModal=(action "onShowPasswordModal")
```

Comment:
In other places it is `Parent::Child` format whereas here it is `parent/child` format. I would suggest to follow the same pattern everywhere for consistency.

Reply:
Parent::Child is valid in octane templating, right? and I think, .emblem won't support this pattern.
please clarify...

Reply:
Didn't notice it is emblem, my bad 😅